.github/workflows/publish.yml

```
@@ -0,0 +1,21 @@
+name: Publish Package to npmjs
+on:
+  release:
+    types: [published]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
```

Can we use v22 please? It's the latest LTS.

```suggestion
          node-version: 22
```

---

Sure thing. I guess that we also upgrade the CI workflow and Docker container base image to use node version 22 too at this point?